challenge2/analysis/AkiOhtani/energy.json

```diff
@@ -0,0 +1,1235 @@
+                                "name": "Mild"
+                            }
+                        ], 
+                        "name": "Mellow"
+                    }, 
+                    {
+                        "children": [
+                            {
+                                "colour": "#efd95f", 
+                                "name": "Nippy"
+                            }, 
+                            {
+                                "colour": "#eee469", 
+                                "name": "Piquant"
+                            }
+                        ], 
+                        "name": "Acidy"
+                    }
+                ], 
+                "name": "Thursday"
+            }, 
+            {
+                "children": [
+                    {
+                        "children": [
+                            {
+                                "colour": "#f6d5aa", 
+                                "name": "Delicate"
+                            }, 
+                            {
+                                "colour": "#f0d48a", 
+                                "name": "Mild"
+                            }
+                        ], 
+                        "name": "Mellow"
+                    }, 
+                    {
+                        "children": [
+                            {
+                                "colour": "#efd95f", 
+                                "name": "Nippy"
+                            }, 
+                            {
+                                "colour": "#eee469", 
+                                "name": "Piquant"
+                            }
+                        ], 
+                        "name": "Acidy"
+                    }
+                ], 
+                "name": "Friday"
+            }, 
+            {
+                "children": [
+                    {
+                        "children": [
+                            {
+                                "colour": "#dbdc7f", 
+                                "name": "Tangy"
+                            }, 
+                            {
+                                "colour": "#dfd961", 
+                                "name": "Tart"
+                            }
+                        ], 
+                        "name": "Winey"
+                    }, 
+                    {
+                        "children": [
+                            {
+                                "colour": "#ebe378", 
+                                "name": "Hard"
+                            }, 
+                            {
+                                "colour": "#f5e351", 
+                                "name": "Acrid"
+                            }
+                        ], 
+                        "name": "Soury"
+                    }
+                ], 
+                "name": "Saturday"
+            }
+        ], 
+        "name": "large"
+    }
+]
```

Comment:
This file is very obscure to me. I expect more energy-related stuff and less "Roasted Peanuts" content.